### ISSUE TITLE
Set the logLevel to appropriate value based on process.env.NODE_ENV

### ISSUE THREAD

**Issue by jahtalab** [elastic/apm-agent-rum-js#639]
Using `process.env.NODE_ENV` we can set the logLevels to an appropriate value by default, this makes the debugging process a bit easier.

**Comment by vigneshshanmugam** [comment]
This is only for dev right?

**Comment by jahtalab** [comment]
Yeah, so `NODE_ENV` is not `production` we should set the log level to debug or trace. (similar to what we do with `__DEV__`)

**Comment by vigneshshanmugam** [comment]
I mean, like do we want to do this for the distribution files or just for our own purposes? 

Doing it for the users/distributions would be a nice addition as well.
